src/computer.cpp

```diff
@@ -369,7 +369,7 @@ void computer::activate_function(computer_action action, char ch)
                                     break;
                                 }
                                 if (!found_item) {
-                                    g->m.add_item_or_charges(x1, y1, sewage);
+                                    g->m.add_item_or_charges( tripoint( x1, y1, g->m.get_abs_sub().z ), sewage );
```

Comment:
Those are good points but almost all of the existing code now calling `map::get_abs_sub` so possibly dropping the last three remaining calls to allow deprecation of the 2D function is worth the small overhead?

What is the consensus?